newbank/server/Account.java

```
@@ -3,22 +3,30 @@
 public class Account {
 	
 	private String accountName;
-	private double openingBalance;
+	private double balance;
 
 	public Account(String accountName, double openingBalance) {
 		this.accountName = accountName;
-		this.openingBalance = openingBalance;
+		this.balance = openingBalance;
 	}
 	
 	public String toString() {
-		return (accountName + ": " + openingBalance + "\n");
+		return (accountName + ": " + balance + "\n");
 	}
 
-	public double getOpeningBalance(){
-		return openingBalance;
+	public double getBalance(){
+		return balance;
 	}
 	public String getAccountName() {
 		return accountName;
 	}
 
+	public void addToBalance(double amountToAdd) {
+		balance = balance + amountToAdd;
+	}
+
+	public void subtractFromBalance(double amountToSubtract) {
+		balance = balance - amountToSubtract;
```

Hey @blnchrdb. I believe it could be worth it adding aguard towards negative amounts. Eg, If a user's balance is like £ 20 and he tries to withdraw or move more than that, the system should return an error?